Makefile

```diff
@@ -94,7 +94,16 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
-
+.PHONY: install-git-hooks
+## Creates useful git hooks (e.g. pre-push go-format-code check)
+install-git-hooks:
+	ln -sfv ../../.pre-push .git/hooks/pre-push
```

Comment:
I want to be able to rename my repo and keep the git hook working. That's why I wanted to a relative link. We recently had to rename the repo or move it somewhere.
I want a link and not some self-contained file because then I can edit the `.pre-push` file in the root folder and immediately test it without having to install it again.